core/src/fr/hammons/slinc/types/Arch.scala

```diff
@@ -9,6 +9,26 @@ private[slinc] enum Arch:
   case X64
   case Unknown
 
+object Arch:
+  extension (arc:Arch)
+    def suffix: String =
+      arc match
+        case I386 => "_i286"
+        case X64 => "_x86_64"
+        case Unknown => ""
+      
+  def inferred(): Arch =
```

Comment:
`os` and `arch` call `System.getProperty`, which causes side effect, so I think method invocation is preferable than `val`.